pandas/__init__.py

```diff
@@ -10,7 +10,7 @@
     try:
         __import__(dependency)
     except ImportError as e:
-        missing_dependencies.append("{0}: {1}".format(dependency, str(e)))
+        missing_dependencies.append(f"{dependency}: {str(e)}")
```

Comment:
Nitpick but no need for the str(e) here can be just `f"{dependency}: {e}"`

Reply:
I was trying to stay consistent with the original code.  I figured that the original author, who is far more experienced than me, had a good reason for using `str(e)`.

Reply:
f strings use __str__ by default?